draft-gharris-opsawg-pcap.md

```diff
@@ -243,11 +243,7 @@ The LinkType Registry is a table of 16-bit numbers.
 The Registry has three sections according to {{RFC8126}}:
 
 * values from 0 to 32767 are marked as Specification Required.
-
-*   except that values 147 to 162 are reserved for Private Use
```

Comment:
I'd leave that in, because not *all* the values in that range are marked as Specification Required.

Perhaps:

* values from 0 to 32767, except for values from 147 to 162, are marked as Specification Required;
* values from 147 to 162 are marked as Private Use;
* values from 32768 to 65000 are marked as First-Come First-Served;
* values from 65000 to 65535 are marked as Private Use.

Alternatively, we could do

* values from 0 to 32767, except for values from 147 to 162, are marked as Specification Required;
* values from 32768 to 65000 are marked as First-Come First-Served;
* values from 147 to 162, and from 65000 to 65535 are marked as Private Use.